### PR TITLE
Honor visibility on custom navigation items

### PR DESCRIPTION
**Pull request by underdpt**
## Description

When using custom navigation items, visibility via`visible()` and/or  `hidden()` is not respected, so all items are always visible.

```php
return $panel
    ->default()
    ->id('config')
    ->navigation(function (NavigationBuilder $builder): NavigationBuilder {
        return $builder->groups([
            NavigationGroup::make('Electricidad')
                ->items([
                    NavigationItem::make('Dashboard')
                        ->visible(false)
                        ->icon('fas-gauge-high')
                        ->url('/dashboard/electricidad'),
                ]),
        ])
        ->items([
            NavigationItem::make('Clientes')
                ->icon('fas-industry')
                ->visible(auth()->user()->can('viewAny', \App\Models\Cliente::class))
                ->url('/admin/clientes'),
            NavigationItem::make('Grupos')
                ->icon('fas-building')
                ->url('/admin/grupos'),
            NavigationItem::make('Incidencias')
                ->icon('fas-circle-exclamation')
                ->url('/admin/incidencias'),
        ]);
    })
```
Items `Dashboard` inside group `Electricidad` and `Clientes` are always visible, but they should't be.

## Code style

- [x] `composer cs` command has been run.

## Testing

- [x] Changes have been tested.

## Documentation

-[x] Documentation is up-to-date.

### CODE REVIEW

**Comment by danharrin** [comment]
Please remove all unrelated formatting changes.

**Comment by underdpt** [comment]
Done. I'm sorry didn't know about that action was ran automatically.

**Comment by elmergustavo** [comment]
thank you ❤

File: packages/panels/resources/views/components/sidebar/group.blade.php [hunk 1]
```diff
@@ -66,6 +66,9 @@ class="fi-sidebar-group-collapse-button"
         class="fi-sidebar-group-items flex flex-col gap-y-1"
     >
         @foreach ($items as $item)
+            @if ($item->isHidden())
+                @continue
+            @endif
```
Comment (zepfietje) [inline packages/panels/resources/views/components/sidebar/group.blade.php]:
```suggestion
            @endif

```

**Pull request merged**